src/types/types.ts

```diff
@@ -0,0 +1,118 @@
+import {TokenData} from "../models/TokenData";
+
+export type TGetUserTokenData = {
+    token?: string;
+    error?: string;
+}
+
+export type TFetchUserDataRes = {
+    token?: string;
+    error?: string;
+}
+
+export type ML_Party = {
+    "party"?: {
```

Comment:
Do you mean, we can have ML_Party as just `{}`? What does it mean all props are optional? Can we avoid mixing camelCase with snake_case naming?
Maybe, it's better to define a separate type for successful and error response:
```
export type MLParty = MLPartySUccessull | MLPartyError;
```